doc/docs/master/reference/pipeline_spec.md

```diff
@@ -410,6 +416,16 @@ datums, and execution time per datum when setting this value. Keep in
 mind that the number of datums may change over jobs. Some new commits may
 have a bunch of new files (and so new datums). Some may have fewer.
 
+### S3 Output Repository
+
+`s3_out` defines whether you want to store the Pachyderm output repository
+in an S3 bucket in an S3 instance deployed as a sidecar in the pipeline
+pod. When this parameter is set to `true`, Pachyderm deploys a sidecar
+container in the same pod as the pipeline container. The address of the
```

Comment:
The sidecar container is always deployed, `s3_out` simply sets whether the sidecar container includes a sidecar s3g instance

Reply:
fixed